### PR TITLE
`use-deliver`: Review contents of exams section

### PR DESCRIPTION
**Pull request by razvand**
Review contents of "Exams" section.

### CODE REVIEW

File: chapters/use-deliver/exams/reading/README.md
Comment (razvand) [reply chapters/use-deliver/exams/reading/README.md]: gotche

File: chapters/use-deliver/exams/reading/README.md
Comment (razvand) [reply chapters/use-deliver/exams/reading/README.md]: roger

File: chapters/use-deliver/exams/reading/README.md
Comment (razvand) [inline chapters/use-deliver/exams/reading/README.md:8]:
```suggestion
- **good content coverage**
```

File: chapters/use-deliver/exams/reading/README.md
Comment (razvand) [inline chapters/use-deliver/exams/reading/README.md:14]:
```suggestion
- **good difficulty coverage**
```

File: chapters/use-deliver/exams/reading/README.md
Comment (razvand) [inline chapters/use-deliver/exams/reading/README.md:12]: Add proper links.

**Review by razvand — COMMENTED** [review]
Rename `Exams` to `Tests` throughout the section. I think it look good overall.

**Comment by Alex-deVis** [comment]
@razvand, this is ready for merge, waiting on @lizababu's approval.

**Review by lizababu — APPROVED** [review]

**Pull request merged**
